<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>ChunxuLi</author>
 <class>loadInfo</class>
 <widget class="QDialog" name="loadInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Object Properties</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>22</y>
        <width>221</width>
        <height>220</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="objName">
           <property name="text">
            <string>objName:</string>
           </property>
           <property name="buddy">
            <cstring>comboBoxName</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="objPose">
           <property name="text">
            <string>objPose:</string>
           </property>
           <property name="buddy">
            <cstring>comboBoxPose</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxPose"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="truncated">
           <property name="text">
            <string>truncated:</string>
           </property>
           <property name="buddy">
            <cstring>comboBoxTruncated</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxTruncated">
           <item>
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="difficult">
           <property name="text">
            <string>difficult:</string>
           </property>
           <property name="buddy">
            <cstring>comboBoxDifficult</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxDifficult">
           <item>
            <property name="text">
             <string>0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="startPointX">
           <property name="text">
            <string>startPoint x:</string>
           </property>
           <property name="buddy">
            <cstring>lineEditStartPointX</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditStartPointX"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="startPointY">
           <property name="text">
            <string>startPoint y:</string>
           </property>
           <property name="buddy">
            <cstring>lineEditStartPointY</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditStartPointY"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="endPointX">
           <property name="text">
            <string>endPoint x:</string>
           </property>
           <property name="buddy">
            <cstring>lineEditEndPointX</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditEndPointX"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="endPointY">
           <property name="text">
            <string>endPoint y:</string>
           </property>
           <property name="buddy">
            <cstring>lineEditEndPointY</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditEndPointY"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>260</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>loadInfo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>loadInfo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
